File: src/frontend/src/app/components/financeiro/caixa/model/caixa-grid-dto.ts
import { Exclude, Expose } from 'class-transformer';

@Exclude()
export class CaixaGridDTO {
  @Expose() id = '';
  @Expose() nome = '';
  @Expose() valorPadraoAbertura = 0;
  @Expose() percentualParcialConfigurado = false;
  @Expose() ativo = true;
  @Expose() unidadeNegocioNome?: string;
  @Expose() deleted = false;
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/financeiro/caixa/CaixaGridDTO.java
package br.com.grupopipa.gestaointegrada.financeiro.caixa;

import java.math.BigDecimal;
import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Getter
@Setter
public class CaixaGridDTO implements GridDTO {
    private UUID id;
    private String nome;
    private BigDecimal valorPadraoAbertura;
    private Boolean percentualParcialConfigurado;
    private Boolean ativo;
    private Boolean deleted;
}
